src/lib/components/Tree/Tree.svelte

```diff
@@ -127,19 +131,35 @@
 				{node.name}
 			</span>
 
-			<button
-				class="text-gray-6"
-				onclick={(event) => {
-					event.stopPropagation()
-					visibility.set(node.id, !isVisible)
-				}}
-			>
-				{#if isVisible}
-					<Eye size={14} />
-				{:else}
-					<EyeOff size={14} />
+			<div class="flex items-center gap-1.5">
+				<button
+					class="text-gray-6"
+					onclick={(event) => {
+						event.stopPropagation()
+						visibility.set(node.id, !isVisible)
+					}}
+				>
+					{#if isVisible}
+						<Eye size={14} />
+					{:else}
+						<EyeOff size={14} />
+					{/if}
+				</button>
+				{#if settings.current.isStandalone}
```

Comment:
I'm concerned that this introduces a lot of UI - and visual noise - for something that won't be done that often. This is also a really valuable real estate area. Can we instead move the delete button to the details panel? cc @mrloureed